_posts/2018/01/2018-01-08-teach-prac-01.md

```diff
@@ -0,0 +1,68 @@
+
+I left the Code till last because it is probably the most important. Unless people observe the Code of Conduct at workshops,
+all our other positive teaching practices can count for nothing. 
+
+Among other things, the Code of Conduct states:
+
+*We are committed to creating a friendly and respectful place for learning, teaching and contributing. 
+All participants in our events and communications are expected to show respect and courtesy to others.*
+
+Instructors introduce the Code of Conduct at the start of workshops for a reason. As a community that values 
+diversity and inclusivity, a community dedicated to providing a welcoming and supportive environment for 
+all people regardless of background or identity, the Code sits at the very heart of everything we do.
+
+If someone breaches the Code in a workshop, the Instructor is empowered to warn that person and, if need be, 
+to have that person removed from the workshop. We also encourage Instructors 
+to [report the behaviour](https://software-carpentry.org/CoC-reporting/) to us. 
+We have developed [a manual](https://software-carpentry.org/CoC-enforcement/) on how to enforce the Code.
+
+Harassment is unacceptable, as the Code clearly states:
+
+*Harassment is any form of behaviour intended to exclude, intimidate, or cause discomfort. 
+Because we are a diverse community, we may have different ways of communicating and of 
+understanding the intent behind actions. Therefore we have chosen to prohibit certain forms of 
+behaviour in our community, regardless of intent.* [Read more.](https://software-carpentry.org/conduct/)
+
+The Code helps people feel safe, which assists their learning. It also makes our workshops 
+accessible to people who might otherwise be marginalised. 
+
+While not as serious as religious, sexual or racial vilification, or the other behaviours we prohibit, 
+there are still many off-putting things that people at workshops can do. If learners 
+are worried about being mocked, talked over, treated with sarcasm, condescended to, or made 
+to feel small or stupid for any reason, their enjoyment of the workshop will be diminished, 
+if not extinguished altogether. In those situations, rather than take the offending person on, 
+some people simply prefer to give up on the workshop, thus losing their opportunity to pick up vital skills.
+
+If they choose to stay, the offence will still take up valuable room in their minds, leaving much less space for learning.
+
+It is therefore up to the Instructors to set the workshop tone. If someone is endlessly parading their knowledge, 
+or hogging workshop time to show off, then the Instructors must try to rein that person in. 
+Your learners will be grateful, and they will also feel you are 'walking the talk', not just paying lip service to an ideal.
+
+A female attendee at a workshop I taught last year wrote on a feedback sticky: "Nice that there are talking rules". 
```

Comment:
I would probably take out "female" here. Partially because feedback stickies are usually anonymous and partially because it implies that talking rules are more important for/more appreciated by women. That last point may or may not be true, but possibly unnecessarily distracting from the point.